### ISSUE TITLE
Implement satellites projects

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#3433]
There was a discussion, at the 41st CGAL developers meeting in Zurich, 2016, about [How to increase academic contributions to CGAL](https://cgal.geometryfactory.com/CGAL/Members/wiki/Meetings/Zurich16/Minutes#How_to_increase_academic_contributions_to_CGAL.3F).

It was proposed that the CGAL project could, for example once a week, a few chosen test external projects using CGAL, in the CGAL testsuite.

The proposal is to start with:
  - https://github.com/Oslandia/SFCGAL
  - https://github.com/CGAL/cgal-swig-bindings

My idea is that is should be sufficient to modify the script that create the internal releases, so that it integrates the code of the external projects in the internal release, or example in `test/xternal_SFCGAL/` (the prefix `xternal_` is there to signal that it is special, and also to push the external package at the end of the alphabetical order).

The directory `test/xternal_.../` would need a special script `cgal_test_with_cmake` that:
  - compiles the software,
  - launches it tests,
  - and creates the right files to be processed by our testsuite tools.

Cc @sloriot @maxGimeno @afabri

**Comment by afabri** [comment]
Wouldn't it make sense to first switch to testing with ctest instead of further complicating our home made scripts?

**Comment by lrineau** [comment]
I do not think so. Most of the difficulty is orthogonal to using our scripts or CTest. I do not think CTest will change anything in the matter of those satellite projects.
